test/test-build.html

```diff
@@ -145,7 +148,7 @@
           return System['import']('amd-6a.js');
         })
         .then(function(m) {
-          ok(window.p == 'a');
+          ok(m.p == 'a');
```

Comment:
It seems that the new wrapping format adds the property p just to the module exports and no longer to the window namespace.

```javascript
"bundle";
System.registerDynamic('amd-6a.js', [], false, function ($__require, $__exports, $__module) {
  return (function (require) {
    this.p = 'a';
  }).call($__exports, $__require);
});
```